### PR TITLE
initial release of rattler-build on conda-forge

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by isuruf** [comment]
Since this borrows code from conda-build, shouldn't that be part of the license?

**Comment by wolfv** [comment]
Hmm, it doesn't really borrow much code from conda-build (except for maybe the environment variables that are available, and how the build script is run). It definitely borrows from conda-build in the docs. Do you have a suggestion regarding what you would want to see in the license?

**Comment by isuruf** [comment]
> Do you have a suggestion regarding what you would want to see in the license?

What I meant was to have https://github.com/conda/conda-build/blob/main/LICENSE in the license for rattler-build

> Hmm, it doesn't really borrow much code from conda-build (except for maybe the environment variables that are available, and how the build script is run)

Not sure if it is needed or not. Ask conda-build maintainers? cc @jezdez, @chenghlee, @kenodegard

**Comment by jezdez** [comment]
> Hmm, it doesn't really borrow much code from conda-build (except for maybe the environment variables that are available, and how the build script is run). It definitely borrows from conda-build in the docs. Do you have a suggestion regarding what you would want to see in the license?

Any code used in the rattler repo coming from conda-build, including the docs, would need to follow the BSD-3-clause license, which is usually just copying the conda-build LICENSE file content to rattler's own LICENSE file, specifying what was copied. That suffice and wouldn't have to be done in the recipe repo at all but uptream.

File: recipes/rattler-build/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# We need fmemopen in libsolv which became available in 10.13
+MACOSX_SDK_VERSION:        # [osx and x86_64]
```
Comment (isuruf) [inline recipes/rattler-build/conda_build_config.yaml:2]: You need `MACOSX_DEPLOYMENT_TARGET` not the SDK unless you are using an availability check.

Comment (wolfv) [reply recipes/rattler-build/conda_build_config.yaml:2]: Thanks!

**Comment by wolfv** [comment]
This should be hopefully good to go now :) 🎸

**Comment by SylvainCorlay** [comment]
Now that the comments have been address, let's go ahead. Congrats on rattler-build @wolfv.

**Pull request merged**